### PR TITLE
UHM-5770: Refactor location tag configuration out of Iniz setup and i…

### PR DESCRIPTION
**Pull request by mogoodrich**
…nto PIH Config

### CODE REVIEW

**Comment by mogoodrich** [comment]
Note that this comment cannot be merged in until the follow Iniz feature gets added:
https://github.com/mekomsolutions/openmrs-module-initializer/issues/137

**Review by cioan — APPROVED** [review]
Looks good!

File: configuration/pih/liquibase/liquibase-haiti.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <comment>
+            UHM-5770: Refactor location tag configuration into Iniz locationtagmaps
+            (One-time clear out of maps so they can be re-initiazed)
+        </comment>
+        <sql>
+            delete from location_tag_map;
+        </sql>
+    </changeSet>
+
+</databaseChangeLog>
```
Comment (mseaton) [inline configuration/pih/liquibase/liquibase-haiti.xml:21]: Per my comment on the other PR, can we avoid this?  It looks like you are defining a large superset of tags in each file, shouldn't this mean that iniz is deleting from location_tag_map as appropriate?  Are there other tags implicated?

**Review by mseaton — APPROVED** [review]
Looks great, nice work.  My only questions here and in the other PR is questioning the need for the liquibase cleanup step and how we add that in.

**Pull request merged**